authorization-api-1_0.md

```diff
@@ -597,9 +600,186 @@ X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305720
   "nextPageToken": "1DlR0Em5panAPy5llasLPfNUpDztEKgTDKF2I5gPwymnc"
 }
 ~~~
-{: #example-search-response title="Example of an Search Response"}
+{: #example-resource-search-response title="Example of a Resource Search Response"}
 
+## Subject Search API
+The Access Subject Search API does the reverse of the Search API: it enables a PEP or client to find out all the subjects that can access a given resource.
 
+The Access Subject Search API is available at the relative URL `/subjectsearch/` via the `POST` HTTP method
+
+### Subject Search Request
+A Subject Search Request has request parameters and a request body. The request parameters are:
+
+pageToken:
+: OPTIONAL. A string value that is returned in a previous Subject Search Response ({{subject-search-response}}), which indicates that the request is a continuation of a previous request
+
+pageSize:
+: OPTIONAL. The maximum number of `decision` items in a Subject Search Response ({{subject-search-response}}). The API MAY return a smaller number of items but SHOULD NOT return a number of items that is greater than this value
+
+The content of a Subject Search Request body is a JSON object with the following fields:
+
+asset:
+: REQUIRED. An asset as described in the Assets section ({{assets}})
+
+queries:
+: REQUIRED. An array of `string` values as described in the Actions section ({{actions}}).
+
+The following is a non-normative example of a Subject Search Request:
+
+~~~ http
+POST /subjectsearch HTTP/1.1
+Host: pdp.mycompany.com?pageToken="NWU0OGFiZTItNjI1My00NTQ5LWEzYTctNWQ1YmE1MmVmM2Q4"&pageSize=2
+Authorization: <myoauthtoken>
+
+{
+  "asset": {
+    "id": "somevalue",
+    "type": "document",
+    "attributeNames": [
+        "author",
+        "createDate",
+        "lastUpdated"
+    ]
+   },
+  "queries": ["delete", "read", "write"]
+}
+~~~
+{: #example-subject-search-request title="Example Subject Search Request"}
+
+### Subject Search Response {#subject-search-response}
+The success response to a Subject Search Request is a Subject Search Response. It is a HTTP response of type `application/json`. Its body is a JSON object that contains the following fields:
+
+iat:
+: REQUIRED. The issued at time in `integer` format, expressed as epoch milliseconds
+
+principal:
+: REQUIRED. The principal for which the response is being issued. The format of this field is as described in the Assets section ({{principals}})
```

Comment:
The subject for which... Also fix reference to {{subjects}}

Reply:
All remaining typos/renames should be done now.